mmcls/models/heads/multi_task_head.py

```diff
@@ -59,22 +70,14 @@ def loss(self, feats: Tuple[torch.Tensor],
             dict[str, Tensor]: a dictionary of loss components
         """
         losses = dict()
-        for task_name, head in self.sub_heads.items():
-            """if 'mask'  in gt_label[task_name].keys()  :
-
-            mask = gt_label[task_name]['mask']
-              label = gt_label[task_name]['label']
-            else: # a tensor
-              label = gt_label[task_name]
-              batch_n = label.shape[0]
-              mask = to_tensor([True]*batch_n)
-            """
+        for task_name, head in self.task_heads.items():
             mask = []
             masked_data_samples = []
             for data_sample in data_samples:
                 sample_mask = data_sample.get_task_mask(task_name)
                 if sample_mask:
-                    sample = data_sample.get_task_sample(task_name)
+                    sample = mmtask_convertor(head.loss, task_name,
```

Comment:
I prefer we do not change the sample here, but directly wrap the head.loss as discussed on slack